### PR TITLE
🪲 [WIP] Failing Test for Comments 

### PR DESCRIPTION
**Pull request by quinnah**
WIP attempt to address #3882

Added two tests test_print_with_hashtag and test_print_another_hashtag in tests/test_level/test_level_04.py

See issue for further comments

### CODE REVIEW

File: tests/test_level/test_level_04.py [hunk 1]
```diff
@@ -88,64 +70,69 @@ def test_print_line_with_spaces_works(self):
...
         code = '''print "It's me"'''
         expected = """print(f'It\\'s me')"""
 
-        self.multi_level_tester(
-            code=code,
-            max_level=11,
-            expected=expected)
+        self.multi_level_tester(code=code, max_level=11, expected=expected)
+
+    def test_print_with_hashtag(self):
```
Comment (quinnah) [inline tests/test_level/test_level_04.py:96]: Relevant test function

**Comment by quinnah** [comment]
Seems like there were issues with autosave/formatting locally which I'll have to resolve ... I added a comment in the relevant test function in the file!

**Comment by Felienne** [comment]
Hi @quinnah!

Thanks a lot for this but at the moment I don't have the bandwidth to help you advance, sorry. Maybe @jpelay can take a look when he has the chance?

**Comment by jpelay** [comment]
Hi, @quinnah! I will be closing this PR for now, because we have a lot of them opened at the moment and we are trying to clean up them. If you want, you can reopen this one in the future to finish it!

**Pull request closed without merging**